### PR TITLE
Templatize

### LINKED ISSUE #8: Add .gitmessage for commit template
Setup git commit message template to automate the commit documentation.

Waiting for #6 to be more solidified.

### PR DESCRIPTION
**Pull request by iboss-ptk**
Resolve #8 

**What kind of change does this PR introduce?**
Add template for commit, PR and issue.
Also introduce setup script for git related:
- setup ETDA/soda-etax as upstream remote
- set `.gitmessage` as commit template

**Does this PR introduce a breaking change?**
no

### CODE REVIEW

File: git-setup.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+# add main repo as an upstream remote so that we can pull lastest from main repo
+echo "* Add main repository as upstream remote"
+git remote add upstream git@github.com:ETDA/soda-etax.git
+
+# add commit template to git config
+echo "* Add git commit guideline into commit template"
+git config --local commit.template .gitmessage
```
Comment (mahasak) [inline git-setup.sh:9]: Can we also provide git-setup for Windows-ish ?

File: .gitmessage [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+# format that includes a **type**, a **scope** and a **subject**. The **header** is mandatory and the **scope** of the header is optional.
+# Any line of the commit message cannot be longer 100 characters! This allows the message to be easier
+# to read on GitHub as well as in various git tools.
+# 
+# ```
+# <type>(<scope>): <subject>
+# <BLANK LINE>
+# <body>
+# ```
+# ### Type
```
Comment (mahasak) [inline .gitmessage:14]: Nice try !!! I love it <3

**Review by mahasak — COMMENTED** [review]
LGTM, Let's wait for another LGTM 😸

**Comment by champillon** [comment]
LGTM also :)

**Review by mahasak — APPROVED** [review]

**Comment by varokas** [comment]
@mahasak or @champillon can you create similar pull requests for all these markdowns (license + how to contribute) to etax-template + other projects?

**Comment by champillon** [comment]
ok let me clone

**Comment by champillon** [comment]
already PR to etax-schema

File: git-setup.sh (hunk 1 above)
Comment (iboss-ptk) [reply git-setup.sh:9]: I don't have windows machine to try that out. Could somebody else do that?

**Pull request merged**